### DISCUSSION TITLE
Color

### DISCUSSION THREAD

**Discussion by EmreCanCakir** [sql-formatter-org/sql-formatter#532] [Q&A]
Is there any color words. For example like this  
![image](https://user-images.githubusercontent.com/83457832/203257958-79e4617e-7781-40b2-a4fe-6df53094f01e.png)

**Comment by nene** [comment]
Sorry. I don't understand the question.

SQL Formatter doesn't do syntax highlighting, if that's what you meant.

**Reply by EmreCanCakir** [reply]
Thanks for answer.

**Comment by maxkremer** [comment]
What is a good option to layer in highlighting along with formatting? Anyone use https://www.npmjs.com/package/sql-highlight in conjunction?
